ambry-api/src/test/java/com.github.ambry/router/InMemoryRouter.java

```diff
@@ -88,7 +89,10 @@ public InMemoryRouter(VerifiableProperties verifiableProperties) {
     private final ByteBuffer blob;
 
     public InMemoryBlob(BlobProperties blobProperties, byte[] userMetadata, ByteBuffer blob) {
-      this.blobProperties = blobProperties;
+      this.blobProperties =
```

Comment:
we create a proper `BlobProperties` here, as this is returned as part of the Get responses (which in reality will have the proper size set). The size here is not used by `readBlob()` (in the Put path)